src/runtime-rs/crates/hypervisor/src/device/driver/virtio_blk.rs

```diff
@@ -152,4 +162,45 @@ impl Device for BlockDevice {
             }
         }
     }
+
+    async fn register_pcie_device(&mut self, topology: &mut PCIeTopology) -> Result<()> {
```

Comment:
The functions `register_pcie_device()` and `unregister_pcie_device()` are similar to `vhost_user_blk.rs`. It is recommended to optimize them.

Reply:
Thx, Fixed